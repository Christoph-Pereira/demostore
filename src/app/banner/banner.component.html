<div class="banner">
    <p class="bannerText">
        This is so beautiful I am going to blame someone else
    </p>
</div>