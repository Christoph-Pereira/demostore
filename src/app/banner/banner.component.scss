.banner {
    background-image: url("../../assets/images/Banner.png");
    min-height: 35.5rem;
}
.bannerText {
    color: white;
    font-size: 2rem;
    padding: 12rem 8rem;
}