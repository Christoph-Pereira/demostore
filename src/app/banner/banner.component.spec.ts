import { ComponentFixture, TestBed } from '@angular/core/testing';

import { BannerComponent } from './banner.component';

describe('BannerComponent', () => {
  let component: BannerComponent;
  let fixture: ComponentFixture<BannerComponent>;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      declarations: [ BannerComponent ]
    })
    .compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(BannerComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });

  it('should contain div with background image', () => {
    const banner = fixture.nativeElement.querySelector('.banner');
    expect(banner).toBeTruthy();
  });
});