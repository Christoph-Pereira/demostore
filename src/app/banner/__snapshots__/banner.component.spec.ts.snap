// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerComponent should match snapshot 1`] = `
<app-banner
  __ngContext__={[Function LRootView]}
>
  <div
    class="banner"
  >
    <p
      class="bannerText"
    >
       This is so beautiful I am going to blame someone else 
    </p>
  </div>
</app-banner>
`;
